2021-08-26 22:37:30,094 INFO    MainThread:56051 [wandb_setup.py:_flush():69] setting env: {'entity': 'rishanthrajendhran', 'project': 'eigengame', 'sweep_id': '4yfv7oi2', 'root_dir': '/Users/rishanthrajendhran/Desktop/EigenGame', 'run_id': 'quco8ztm', 'sweep_param_path': '/Users/rishanthrajendhran/Desktop/EigenGame/wandb/sweep-4yfv7oi2/config-quco8ztm.yaml'}
2021-08-26 22:37:30,094 INFO    MainThread:56051 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 22:37:30,094 INFO    MainThread:56051 [wandb_init.py:_log_setup():336] Logging user logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20210826_223730-quco8ztm/logs/debug.log
2021-08-26 22:37:30,095 INFO    MainThread:56051 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20210826_223730-quco8ztm/logs/debug-internal.log
2021-08-26 22:37:30,095 INFO    MainThread:56051 [wandb_init.py:init():369] calling init triggers
2021-08-26 22:37:30,095 INFO    MainThread:56051 [wandb_init.py:init():376] wandb.init called with sweep_config: {'learningRate': 1e-06}
config: {'xDim': (100000, 4), 'numIterations': 10000, 'k': 4, 'learningRate': 1}
2021-08-26 22:37:30,095 INFO    MainThread:56051 [wandb_init.py:init():418] starting backend
2021-08-26 22:37:30,095 INFO    MainThread:56051 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 22:37:30,114 INFO    MainThread:56051 [backend.py:ensure_launched():132] starting backend process...
2021-08-26 22:37:30,123 INFO    MainThread:56051 [backend.py:ensure_launched():137] started backend process with pid: 56061
2021-08-26 22:37:30,132 INFO    MainThread:56051 [wandb_init.py:init():423] backend started and connected
2021-08-26 22:37:30,141 INFO    MainThread:56051 [wandb_run.py:_config_callback():787] config_cb None None {'learningRate': 1e-06}
2021-08-26 22:37:30,142 INFO    MainThread:56051 [wandb_init.py:init():465] updated telemetry
2021-08-26 22:37:30,144 INFO    MainThread:56051 [wandb_init.py:init():484] communicating current version
2021-08-26 22:37:31,175 INFO    MainThread:56051 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-26 22:37:31,175 INFO    MainThread:56051 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-08-26 22:37:31,560 INFO    MainThread:56051 [wandb_init.py:init():522] starting run threads in backend
2021-08-26 22:37:36,571 INFO    MainThread:56051 [wandb_run.py:_console_start():1565] atexit reg
2021-08-26 22:37:36,571 INFO    MainThread:56051 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-08-26 22:37:36,572 INFO    MainThread:56051 [wandb_run.py:_redirect():1417] Redirecting console.
2021-08-26 22:37:36,574 INFO    MainThread:56051 [wandb_run.py:_redirect():1479] Redirects installed.
2021-08-26 22:37:36,574 INFO    MainThread:56051 [wandb_init.py:init():546] run started, returning control to user process
2021-08-26 22:39:11,122 INFO    MainThread:56051 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-08-26 22:39:11,124 INFO    MainThread:56051 [wandb_run.py:_restore():1507] restore
2021-08-26 22:39:13,137 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 777
}

2021-08-26 22:39:13,550 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 777
}

2021-08-26 22:39:13,864 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 15284
}

2021-08-26 22:39:14,417 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:14,523 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:14,628 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:14,733 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:14,838 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:14,944 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:15,048 INFO    MainThread:56051 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15284
  total_bytes: 15284
}

2021-08-26 22:39:16,431 INFO    MainThread:56051 [wandb_run.py:_show_files():1879] logging synced files
